pulp_ansible/tests/functional/api/collection/test_marks.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from pulp_smash.pulp3.bindings import monitor_task
+from pulp_smash.pulp3.utils import (
+    get_added_content_summary,
+    get_content,
+    get_content_summary,
+    gen_distribution,
+    get_removed_content_summary,
+)
+from pulp_ansible.tests.functional.utils import (
+    gen_ansible_remote,
+)
+from pulpcore.client.pulp_ansible import AnsibleRepositorySyncURL
+
+
+def test_add_mark_to_collection_version(
+    build_and_upload_collection,
+    ansible_repo_api_client,
+    ansible_repo,
+    ansible_collection_mark_client,
+):
+    """Test adding a mark to a collection version.
+
+    1. Build and upload a collection
+    2. Add it to a repo version
+    3. Add a mark to the collection version
+    4. Assert the mark was added correctly
+    5. Remove the mark
+    6. Assert the mark was removed correctly
+    """
+    collection, collection_url = build_and_upload_collection()
+
+    # add it to a repo version
+    body = {"add_content_units": [collection_url]}
+    monitor_task(ansible_repo_api_client.modify(ansible_repo.pulp_href, body).task)
```

Comment:
monitor_task is available as a fixture now.

Anyway, Yay for writing pytest test!